public/404.html

```diff
@@ -1,2 +1,2 @@
 <!-- Redirect back to homepage -->
-<meta http-equiv="refresh" content="0; url = https://primer.style/prism" />
+<meta http-equiv="refresh" content="0; url = /prism" />
```

Comment:
This would redirect to Primer's homepage whenever Discord live-reloads... not entirely sure why this fixes it.